runtime/convert.c

```diff
@@ -0,0 +1,16 @@
+#include "balrt.h"
+
+TaggedPtr _bal_tagged_to_tagged_float(TaggedPtr tp) {
```

Comment:
Let's call this `_bal_convert_to_float`.

Reply:
Changed